spec/requests/api/collections_spec.rb

```diff
@@ -84,6 +84,11 @@ def test_collection_bulk_query(collection, collection_url, klass, id = nil)
       test_collection_query(:clusters, clusters_url, EmsCluster)
     end
 
+    it "query CloudVolumes" do
+      FactoryGirl.create(:cloud_volume)
+      test_collection_query(:cloud_volumes, cloud_volumes_url, CloudVolume)
+    end
+
```

Comment:
also probably need just the resource gets tests, look at spec/requests/api/regions_spec.rb and create a spec/requests/api/cloud_volumes_spec.rb